napari/layers/_shapes_layer/model.py

```diff
@@ -1183,6 +1184,46 @@ def _move(self, coord):
                 self._drag_box = np.array([self._drag_start, coord])
                 self._set_highlight()
 
+    def to_svg(self, canvas_shape=None, shape_type=None):
+        """Returns an svg string with all the shapes contained in the layer.
+        Passing a `shape_type` argument leads to only shapes from that
+        particular `shape_type` being returned.
+
+        Parameters
+        ----------
+        canvas_shape : tuple, optional
+            2 tuple defining shape of svg canvas to be generated. If not
+            specified, takes the max of all the vertiecs
+        shape_type : {'line', 'rectangle', 'ellipse', 'path', 'polygon'} |
+                     None, optional
```

Comment:
again, don't use `| None` when you have `, optional`

Reply:
fixed